### ISSUE TITLE
9. Images, Art & Video - challenge

### ISSUE THREAD

**Issue by JunsolKim** [UChicago-Computational-Content-Analysis/Readings-Responses-2023#4]
Post your response to our challenge questions.

What image or audio data could be relevant to your course/thesis/life research question(s)? What intuitions do you have about the broad content patterns you would express across and between text, image, and/or audio data? Could pre-trained models allow you to unleash image or audio features that would allow you to parsimoniously test this to support your project?

**Comment by konratp** [comment]
Because the speeches given in parliament are so meticulously documented, I'm struggling to think of ways in which my final project (which utilizes textual data from parliamentary speeches) could be enhanced by image or audio data. I don't know if this is possible yet, but perhaps analyzing the sentiments of a speech not just based on its content but also its delivery could be interesting. Do people's voices become shaky more often when they speak about topics that hit closer to home? Do politicians strategically tap into their emotions when delivering a speech, and can we detect that by picking up on changes in how their voices sound? I think there could be interesting analyses conducted about this, but I'm not sure if I'm personally ever going to be trained well enough to do it myself.

**Comment by GabeNicholson** [comment]
A random one I have is about comedy.

It's a notoriously difficult subject to study and even define for us humans. What makes something funny? The usual answer is "I don't know it just was funny." And comedy goes beyond word-to-word translation which is why the saying "You had to be there to get it," is saying something about the delivery of comedy that goes beyond simply repeating what was said verbatim. 

I am curious about what audio data could reveal about comedy (coming from a neural network) and then reverse-engineering the architecture of the neural network to find out what makes something funny. This is completely theoretical and abstract, but something I find interesting nonetheless. The first step, I think, would be to classify an audio clip as funny or not funny and go from there.

**Comment by facundosuenzo** [comment]
I'm working with a data set of news to explore the relationship between technology and imaginaries of the future. Thus, it'd be interesting to see how the images that usually accompany news articles either anchor or relay (in terms of Roland Barthes) particular emotions to convey or assert meaning or a direction. 
I'd probably start classifying a subset of images and then extrapolating and testing with the bulk of images.

**Comment by ValAlvernUChic** [comment]
I asked this in the first week but it seems most relevant here -  
_"While this assigned paper was largely focused on text-based data, I couldn't help but think about memes as a communicative platform (in some of my circles it's the main communication method, unfortunately). Intuitively, negotiating textual cues with the often contextually-saturated images that they caption would seem extremely difficult. At the same time, in a digital age when these mediums increasingly dominate political and social discourse, they seem incredibly important to theorize. A paper by Dimitrov et al., "Detecting Propaganda Techniques in Memes", did something interesting but they still relied on manual annotations of memes before using them. I was wondering about 1) the advances in methodological approaches in this multi-modal space - specifically whether anyone has had success with unsupervised approaches and 2) How we might even approach this if the context is so latent - I know why Will Smith is crying in that photo and the type of text most appropriate for it, but how can we get the computer to know too?"_

Basically the same question - are there methodologies that can integrate the text in memes and the actual latent context found in the images?

**Comment by ValAlvernUChic** [comment]
It'd be interesting to get audio recordings of members of parliament talking about transient workers and matching those speeches, along with the accompanying tone and cadence in their voice, for additional analysis of the actual textual parliament text. From this, we can get more insight into micro-aggressions or micro-emotions expressed.

**Comment by hsinkengling** [comment]
I'm interested in news photos. I wonder if we can analyze news photos and extract features that help us determine what effect do the different photos used in news articles affect their click rates, how the articles and headlines are interpreted? (as measured in the comments)

**Comment by shosamane** [comment]
I am really interested to know what makes a good orator - the tone intonations, the pauses, choice of words, the fervour and see if I can map these attributes to populist leaders. I would try to see the feasibility of using these deep learning models to classify if a speaker is persuasive. I would start collecting speeches that have previously caused riots (Jan 6th, John Lewis incident) or election rally speeches in contested states where the candidate later won (not focussing on causality though).

**Comment by NaiyuJ** [comment]
To better understand ethnic minorities in China, we could analyze the photos that people from different ethnic minority groups posted on their social media to see whether specific ethnic groups, especially those with small populations, are more likely to dress up with cultural characteristics.

**Comment by isaduan** [comment]
Some data I am interested in: 

1. whether we could train AI to predict winners of Presidential debates or just normal debates
2. to what extents are international politics round tables and panel discussions, now frequently recorded and posted on YouTube, productive discussions (e.g. do they predict the development of current affairs, like war in Ukraine, or are most of them vague, uninformative, performative speeches)? how does seniority and power affect conversations?

**Comment by Jiayu-Kang** [comment]
I don't have very specific thoughts, but in general I'm curious about: (1) the relationship between the poster/audio of movies and their genres (2) how the poster/audio of movies predict its scores and reviews.

**Comment by pranathiiyer** [comment]
This is something that one of institutes on campus is currently researching but I guess I'd be really curious to delve deeper into it. I wonder how audio recordings of conversations between police officers change as they interact with each other before or during criminal events or encounters of any sort? More so because this is the only kind of data that actually captures interaction if this kind.

**Comment by Qiuyu-Li** [comment]
My final project is about the misogyny discourse on Twitter. My current plan is to construct a lexicon to retrieve misogyny tweets, but while conducting it, I found myself ignoring important pieces of information: the images and videos posted by Twitter users. I believe taking these images and videos as part of the feature sets of misogyny classifier training would significantly improve the accuracy.

**Comment by YileC928** [comment]
I am interested in investor and analyst behavior on social media. A data source I think might be helpful is Youtube - thumbnail images, audios, and videos. Variables such as image objects, gender, voice pitch, speech content could be extracted and used to explore questions such as what drives attention and engagement, and do male and female analysts behave similarly.

**Comment by LuZhang0128** [comment]
Since I'm interested in the #BLM movement, it would be interesting if I can get some recordings or pictures of the in-person protests. Using pre-trained models, one simple thing to do is to see if males' and females' speeches are different in terms of mention (stress on certain words or syllables). Also it would be interesting to do some facial analyze of emotions based on pictures

**Comment by mikepackard415** [comment]
I have a budding interest in gardening (pun intended), and there's an app I use that identifies plants from pictures and can suggest how to care for them. I wonder if there are extensions from that basic idea that could be even more useful for gardening or urban food forest purposes.

**Comment by Jasmine97Huang** [comment]
Lyrics are only one of many forms of artists expressions that reflects cultural shifts. It would be interesting to combine analysis of popular cultural figures fashion or styles though images with lyrical understandings.

**Comment by hshi420** [comment]
I'm interested in news sentiment difference when reporting news about different countries, so I can analyze facial expression of the newscasters. I would expect they to have positive expressions when reporting news good to their countries, vice versa.

**Comment by chentian418** [comment]
I am interested in the audios in earnings conference calls. Audios contain more sentimental information than semantical words, and by conducting voice analysis on managers' speeches in the Q&A section of conference calls, and this convey incremental informativeness to the markets. By using the GMM-HMM model and pre-training using the historical speeches of the same manager, I hope to uncover the acoustic features and phonemes behind the scenes.

**Comment by Hongkai040** [comment]
My final project is movie comments sentiment Analysis. In many cases those comments are just reiterating lines from movies. These lines have contexts. The way these actors deliver the lines could influence the textual meaning of those lines. So I think using pre-trained models to analyze the sentiment in audios and videos where the actors deliver their lines could help improve the analysis of movie comments.

**Comment by Emily-fyeh** [comment]
I am interested in the transformation of political memes. The meanings and inferences of memes are highly subject to the context, social environment, and recent events. People from different subcultures may use the same meme material differently. By vectorizing the pixels of the meme pictures and processing the word sequences on the picture, I think it will be very promising to have a model predicting the context given a meme, or to generate new memes from given materials.

**Comment by ZacharyHinds** [comment]
As I am interested in online extremism, specifically looking at the Incels, I am interested if there is anything to be gained from doing image analysis on users' profile pictures? It seems unlikely, but I would still be interested to investigate that. Furthermore, users on the Incels forum I pulled my data from often post images, but again I struggle to identify specifically how I could best analyze those images to contribute to my understanding of the Incels extremist community.

**Comment by ttsujikawa** [comment]
As I am leaning toward researching how people in a diverse setting build their relationship, I think that audio of conversation would help me go deeper into analyzing their sentiment and its transition. If, with pre-trained model, we could know detailed sentiment, it could help my understanding of correlation between people's network and their sentiment.

**Comment by sizhenf** [comment]
Some social media platform allows users to attach pictures in their posts and comments, and thus they may carry additional information other than the text.

**Comment by chuqingzhao** [comment]
To understand the storytelling for pitch behavior, I think the video and audio data from pitch competition could be an ideal resource for sentiment, pitch pattern analysis.
